db/migrate/20220322165424_create_posts.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :body
+      t.references :user, null: false, foreign_key: true
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- The post and user migrations or making associations were not in this milestone. Thanks for the extra work but always work according to your milestone need. :+1:

Reply:
Hi @Dipeshtwis, I was having my doubts about this but decided to give it a try 😆 I have now deleted the migrations and altered the HTML files to only display dummy text.